test/filters_test.py

```diff
@@ -201,3 +201,48 @@ def test_fail_wrong_type(self):
       [TryteString(b'RBTC9D9DCDQAEASBYBCCKBFA')],
       [f.Type.CODE_WRONG_TYPE],
     )
+
+
+# noinspection SpellCheckingInspection
+class AddressNoChecksumTestCase(BaseFilterTestCase):
+  filter_type = AddressNoChecksum
+
+  # noinspection SpellCheckingInspection
+  def setUp(self):
+    super(AddressNoChecksumTestCase, self).setUp()
+
+    # Define some addresses that we can reuse between tests
+    """
+    Incoming value is not an :py:class:`Address` instance.
+    """
+    self.tryte1 = (
+      b'TESTVALUE9DONTUSEINPRODUCTION99999FBFFTG'
+      b'QFWEHEL9KCAFXBJBXGE9HID9XCOHFIDABHDG9AHDR'
+    )
+    self.checksum = b'ENXYJOBP9'
+    self.address = Address(self.tryte1)
+    self.address_with_checksum = Address(self.tryte1 + self.checksum)
+    self.address_with_bad_checksum = Address(self.tryte1 + b'DEADBEEF9')
+
+  def test_pass_checksumless_addy(self):
```

Comment:
😸 Not to be confused with addresses that have multiple checksums — checksummore addys :P